<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">

QPushButton,QLabel {
    background: transparent; /* Make sure the buttons and labels are transparent */
    border: none; /* Remove any borders from buttons or labels */
    color: #FFFFFF; /* White text for contrast */
}

QStackedWidget {
    background-color: #FFEB3B;  /* Bright yellow background */
    color: rgb(0, 0, 0);  /* Black text for high contrast */
    border: 10px solid #000000;  /* Black border for the page */
    border-radius: 15px;  /* Rounded corners for the page */
    padding: 20px;  /* Padding inside the border */
    width: 100%;  /* Ensure it spans the full width */
    height: 100%;  /* Ensure it spans the full height */
    box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.3);  /* Darker shadow for more depth */
}

QPushButton {
    background-color: #FFC107;  /* Yellow background for buttons */
    color: #000000;  /* Black text for buttons */
    border: 2px solid #000000;  /* Black border around the button */
    border-radius: 10px;  /* Rounded corners for buttons */
    padding: 12px 24px;  /* Add padding for button size */
    font-weight: bold;  /* Bold text */
    font-size: 16px;  /* Slightly larger font for buttons */
    cursor: pointer;  /* Change cursor to pointer on hover */
    transition: background-color 0.3s, transform 0.2s, box-shadow 0.3s;  /* Smooth transition for hover effect */
}

QPushButton:hover {
    background-color: #FF9800;  /* Slightly darker yellow for hover */
    transform: scale(1.05);  /* Slightly enlarge the button */
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);  /* Darker shadow on hover */
}

QPushButton:pressed {
    background-color: #FF5722;  /* Darker orange when pressed */
    transform: scale(0.95);  /* Slightly shrink the button when pressed */
    box-shadow: none;  /* Remove shadow when pressed */
}

QPushButton:focus {
    outline: none;  /* Remove default outline */
    box-shadow: 0 0 8px rgba(0, 0, 0, 0.6);  /* A glowing outline on focus */
}






</string>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="Welcome_Label">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: #FFEB3B;

font-size: 24px; 
 font-weight: bold;



</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="text">
             <string>✨   Welcome to Conversa!   ✨</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Play_Button">
            <property name="styleSheet">
             <string notr="true">QPushButton#backButton {
    color: rgb(0, 0, 0);
background-color: #FFEB3B;
border: 2px solid #000000;  
border-radius: 12px;  /* Rounded corners */
 padding: 10px 20px;
font-size: 16px; 
 font-weight: bold;
    transition: background-color 0.3s, transform 0.2s, color 0.3s;  /* Smooth transition for hover effect */
}

QPushButton#backButton:hover {
    background-color: #FFC107;  /* Lighter yellow when hovering */
    color: white;  /* White text when hovered */
    transform: scale(1.05);  /* Slightly enlarge the button when hovered */
}

QPushButton#backButton:pressed {
    background-color: #FF9800;  /* Darker yellow when the button is pressed */
    transform: scale(0.95);  /* Slightly shrink the button when pressed */
}

QPushButton#backButton:focus {
    outline: none;  /* Remove the default focus outline */
    box-shadow: 0 0 5px rgba(255, 193, 7, 0.8);  /* Add a glowing outline for focus state */
}</string>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="Spanish_Button">
            <property name="styleSheet">
             <string notr="true">border-color: 2px solid qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));</string>
            </property>
            <property name="text">
             <string>Spanish</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Galician_Button">
            <property name="text">
             <string>Galacian</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Catalan_Button">
            <property name="text">
             <string>Catalan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="SerboCroatian_Button">
            <property name="text">
             <string>Serbo-Croatian</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="BackButton1">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">QLabel{
    background-color: #FFC107; /* Golden background */
    color: #000000; /* Black text */
    font-family: 'Comic Sans MS', sans-serif; /* Funky font */
    font-size: 24px; /* Larger font size */
    text-align: center; /* Center-align the text horizontally */
    vertical-align: middle; /* Center-align the text vertically */
    padding: 20px;
    border-radius: 15px;
    box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.2); /* Shadow effect */
    font-weight: bold; /* Make it bold for emphasis */
    height: 100px; /* Ensure label height to center vertically */
}


</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>spain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="translateLabel">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Translaion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="WordOfDay_Label">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Word of the day</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="BackButton2">
            <property name="styleSheet">
             <string notr="true">QPushButton#backButton {
    background-color: #FFEB3B;  /* Bright yellow background */
    color: rgb(10, 10, 10);  /* Dark gray text color for good contrast */
    border: 2px solid #FFC107;  /* Yellow border for the button */
    border-radius: 10px;  /* Rounded corners for the button */
    padding: 10px 20px;  /* Padding inside the button */
    font-size: 16px;  /* Font size */
    font-weight: bold;  /* Bold text for emphasis */
    cursor: pointer;  /* Change cursor to pointer when hovering */
    transition: background-color 0.3s, transform 0.2s, color 0.3s;  /* Smooth transition for hover effect */
}

QPushButton#backButton:hover {
    background-color: #FFC107;  /* Lighter yellow when hovering */
    color: white;  /* White text when hovered */
    transform: scale(1.05);  /* Slightly enlarge the button when hovered */
}

QPushButton#backButton:pressed {
    background-color: #FF9800;  /* Darker yellow when the button is pressed */
    transform: scale(0.95);  /* Slightly shrink the button when pressed */
}

QPushButton#backButton:focus {
    outline: none;  /* Remove the default focus outline */
    box-shadow: 0 0 5px rgba(255, 193, 7, 0.8);  /* Add a glowing outline for focus state */
}
</string>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Quiz_Button">
            <property name="styleSheet">
             <string notr="true">QPushButton#backButton {
    background-color: #FFEB3B;  /* Bright yellow background */
    color: rgb(10, 10, 10);  /* Dark gray text color for good contrast */
    border: 2px solid #FFC107;  /* Yellow border for the button */
    border-radius: 10px;  /* Rounded corners for the button */
    padding: 10px 20px;  /* Padding inside the button */
    font-size: 16px;  /* Font size */
    font-weight: bold;  /* Bold text for emphasis */
    cursor: pointer;  /* Change cursor to pointer when hovering */
    transition: background-color 0.3s, transform 0.2s, color 0.3s;  /* Smooth transition for hover effect */
}

QPushButton#backButton:hover {
    background-color: #FFC107;  /* Lighter yellow when hovering */
    color: white;  /* White text when hovered */
    transform: scale(1.05);  /* Slightly enlarge the button when hovered */
}

QPushButton#backButton:pressed {
    background-color: #FF9800;  /* Darker yellow when the button is pressed */
    transform: scale(0.95);  /* Slightly shrink the button when pressed */
}

QPushButton#backButton:focus {
    outline: none;  /* Remove the default focus outline */
    box-shadow: 0 0 5px rgba(255, 193, 7, 0.8);  /* Add a glowing outline for focus state */
}
</string>
            </property>
            <property name="text">
             <string>Quiz</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Flashcards_Button">
            <property name="styleSheet">
             <string notr="true">QPushButton#backButton {
    background-color: #FFEB3B;  /* Bright yellow background */
    color: rgb(10, 10, 10);  /* Dark gray text color for good contrast */
    border: 2px solid #FFC107;  /* Yellow border for the button */
    border-radius: 10px;  /* Rounded corners for the button */
    padding: 10px 20px;  /* Padding inside the button */
    font-size: 16px;  /* Font size */
    font-weight: bold;  /* Bold text for emphasis */
    cursor: pointer;  /* Change cursor to pointer when hovering */
    transition: background-color 0.3s, transform 0.2s, color 0.3s;  /* Smooth transition for hover effect */
}

QPushButton#backButton:hover {
    background-color: #FFC107;  /* Lighter yellow when hovering */
    color: white;  /* White text when hovered */
    transform: scale(1.05);  /* Slightly enlarge the button when hovered */
}

QPushButton#backButton:pressed {
    background-color: #FF9800;  /* Darker yellow when the button is pressed */
    transform: scale(0.95);  /* Slightly shrink the button when pressed */
}

QPushButton#backButton:focus {
    outline: none;  /* Remove the default focus outline */
    box-shadow: 0 0 5px rgba(255, 193, 7, 0.8);  /* Add a glowing outline for focus state */
}
</string>
            </property>
            <property name="text">
             <string>Flashcards</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">QLabel{
    background-color: #FFC107; /* Golden background */
    color: #000000; /* Black text */
    font-family: 'Comic Sans MS', sans-serif; /* Funky font */
    font-size: 24px; /* Larger font size */
    text-align: center; /* Center-align the text horizontally */
    vertical-align: middle; /* Center-align the text vertically */
    padding: 20px;
    border-radius: 15px;
    box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.2); /* Shadow effect */
    font-weight: bold; /* Make it bold for emphasis */
    height: 100px; /* Ensure label height to center vertically */
}</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="quizQuestionLabel">
          <property name="styleSheet">
           <string notr="true">alternate-background-color: rgb(232, 164, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="quizOption3">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="quizOption4">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="quizOption1">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="quizOption2">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>RadioButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="quizNextButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BackButton3">
          <property name="text">
           <string>Exit Quiz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="quizFeedbackLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="quizFinalScoreLabel">
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>                                                                                            TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="endQuizButton">
            <property name="text">
             <string>Go back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="ImageLabel">
            <property name="minimumSize">
             <size>
              <width>811</width>
              <height>160</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid #000000</string>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="foreignWordLabel">
                <property name="styleSheet">
                 <string notr="true">background-color: #FFC107;  /* Yellow background for buttons */
    color: #000000;  /* Black text for buttons */
    border: 2px solid #000000</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Shape::NoFrame</enum>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="englishWordLabel">
                <property name="styleSheet">
                 <string notr="true">background-color: #FFC107;  /* Yellow background for buttons */
    color: #000000;  /* Black text for buttons */
    border: 2px solid #000000</string>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="flipButton">
                <property name="text">
                 <string>FLIP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="nextFlashcardButton">
                <property name="text">
                 <string>NEXT</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BackButton4">
            <property name="text">
             <string>Exit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
